hw/top_earlgrey/data/ip/chip_sysrst_ctrl_testplan.hjson

```diff
@@ -18,6 +18,10 @@
       si_stage: SV3
       lc_states: ["DEV", "PROD", "PROD_END", "RMA"]
       tests: ["chip_sw_pwrmgr_sysrst_ctrl_reset", "chip_sw_sysrst_ctrl_reset"]
+      bazel: [
+        "//sw/device/tests:pwrmgr_deep_sleep_all_reset_reqs_test",
+        "//sw/device/tests:pwrmgr_normal_sleep_all_reset_reqs_test",
```

Comment:
Are you sure these two tests definitely cover the test plan above?

It's not obvious from the C file that they do the steps listed in the description. Are these things happening behind the scenes?

Reply:
This is done in [the pwrmgr reset library](https://github.com/lowRISC/opentitan/blob/master/sw/device/tests/pwrmgr_sleep_resets_lib.c#L181). The function `config_sysrst` is called in both tests and configures the sysrst block to reset when a specific combo is detected.

Reply:
Aside, should that C file move to `//sw/device/lib/testing/...` with the other testutils, or is it different in some way?

Reply:
I think it's a little bit too specific to be in testutils. Basically there are four very similar tests that involve doing resets in mutiple different ways (pwrmgr, sysrst, alertmgr and wdog). This library is there to configure those 4 sources and factor out the code of those 4 tests. But pretty much everything in this library is specific to those tests.

I think it should be the other way around: those four tests should probably have only one C file that uses one or two defines to change the maybe one or two things that differ between them.